samples/library/DeleteDocument.js

```diff
@@ -0,0 +1,34 @@
+const mstr = require('../../lib/mstr');
+
+(async () => {
+  const baseUrl = 'http://10.23.1.124:8080/MicroStrategyLibrary/api';
+  const mstrApi = new mstr.REST({
+    baseUrl: baseUrl,
+  });
+
+  await mstrApi.login({
+    username: 'Administrator',
+    password: '',
+    loginMode: 1,
+  });
+
+  // MicroStrategy Tutorial
+  const projectId = 'B19DEDCC11D4E0EFC000EB9495D0F44F';
+  mstrApi.setProjectId(projectId);
+
+  //Document id:
+  const objectId = '51BF187841C48CD995DFC4A853941F13';
+
+  const libraryAPI = mstrApi.library;
+
+  try {
+    console.log('Deleting object from Library');
+    const result = await libraryAPI.deleteObject(projectId, objectId);
```

Comment:
This is what I mean with making projectID a latter parameter. If you're using `setProjectId` (line 17), the lib can automatically read the project ID from there, unless you really want to provide it in your deleteObject() API call. See some of the other samples for a demo if you want to see it in action.

Reply:
This would essentially clean the function call into this, since projectId becomes optional if already set on a higher level:
```const result = await libraryAPI.deleteObject(objectId);```

Reply:
Vale! :)